src/__main__.py

```diff
@@ -88,7 +88,7 @@ def main():
         vertex_indices, edge_indices = graph.pcsf()
         forest, augmented_forest = graph.output_forest_as_networkx(vertex_indices, edge_indices)
 
-    output_networkx_graph_as_graphml_for_cytoscape(augmented_forest, args.output_dir)
+    #output_networkx_graph_as_graphml_for_cytoscape(augmented_forest, args.output_dir)
```

Comment:
This function wouldn't run because of an issue with networkx and numpy values, that I see you've commented on before (https://github.com/networkx/networkx/issues/2702)